src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false)
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private String pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book() {
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
This setter should not be included because we don't want the id to be modified from outside this class 🙂